media-libs/libpng/libpng-1.5.26.recipe

```diff
@@ -2,16 +2,16 @@ SUMMARY="Portable Network Graphics library"
 DESCRIPTION="libpng is the official PNG reference library. It supports almost \
 all PNG features, is extensible, and has been extensively tested for over 17 \
 years"
-HOMEPAGE="http://www.libpng.org"
-COPYRIGHT="1998-2011 Glenn Randers-Pehrson
+HOMEPAGE="http://www.libpng.org/"
+COPYRIGHT="1998-2015 Glenn Randers-Pehrson
 	   1996-1997 Andreas Dilger
 	   1995-1996 Guy Eric Schalnat, Group 42, Inc."
 LICENSE="LibPNG"
-REVISION="2"
-SOURCE_URI="http://prdownloads.sourceforge.net/libpng/libpng-$portVersion.tar.gz"
-CHECKSUM_SHA256="1e25d7937ecda0bad46ab1b70f6f906c86efefc365a6f6c97ae6ae3895b17aa3"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/project/libpng/libpng15/$portVersion/libpng-$portVersion.tar.xz"
```

Comment:
Is there a reason you changed shorter link to a longer one?
Even though 80-character limit rule does not apply to links, it is preferred for them to follow it.
It can be even shorter: `http://prdownloads.sf.net/libpng/libpng-$portVersion.tar.xz`